plugin/vscode/BUILDING.md

```diff
@@ -13,6 +14,8 @@ and [Node.js][], then run the following commands:
     $ emmake ninja -C build-emscripten quick-lint-js-vscode quick-lint-js-vscode-licenses
     $ emmake cmake --install build-emscripten --component vscode --prefix .
 
+**Note for Windows Users**: In the [CMakeLists.txt](https://github.com/lifeinData/quick-lint-js/blob/master/plugin/vscode/CMakeLists.txt) file,  remove the apostrophe character before "[" and "]" in [line 14](https://github.com/lifeinData/quick-lint-js/blob/d0d48525151101a46627c454618870cb7d6dbb2c/plugin/vscode/CMakeLists.txt#L14)
```

Comment:
Let's codify this suggestion in the file itself, rather than telling people to change files. This suggestion could get out of date.